### DISCUSSION TITLE
Least favourite MFX.

### DISCUSSION THREAD

**Discussion by bangcorrupt** [bangcorrupt/hacktribe#254] [Q&A] [answered]
Which MFX presets could you live without?

I have a couple of reverbs, a stereo tape delay and tube saturation I'd like to add to Hacktribe MFX.  For the time being I can't add new MFX, only replace existing ones.  Post the MFX you use least, if nobody agrees I'll add them to a poll.

I never use Autopan or Jag Filter.

**Comment by enorrmann** [comment]
for me that would be
Wet Reverb
Vinyl Break
Auto Pan

related question, can the Reverb be added as an IFX instead ?

**Reply by bangcorrupt** [reply, marked as answer]
https://github.com/bangcorrupt/hacktribe/discussions/174#discussioncomment-2989455

Maybe some day we will have full control over the audio routing, but reverb as a single channel insert doesn't make much sense in a mix, to me at least.

**Comment by BKLronin** [comment]
I only use all the reverbs and the compressor.

**Reply by tripsylon** [reply]
Autopan, vinylbreak, the filters... 99% of the time I use the mod delay, sometimes chorus and the shifters ... I was wondering would it be possible to manipulate the output compression outside the mfx, somewhere in the system  ? To have the option to compres the sound and use some nice stereo fx at the same time… a stereo reverb for the IFX would be a nice add for me…

**Reply by BKLronin** [reply]
I would specualte that reverb would eat up a lot of voices in the ifx. The ones in the MFX seems to be reserved.

**Reply by tripsylon** [reply]
You are probably right but sometimes it would add  nice spice to the  mix 😊…

**Reply by BKLronin** [reply]
Yes indeed, I do like to reverberate per part too but mostly I leave stuff dry and hand that over to the daw. Love the export function on the E2.

**Comment by tripsylon** [comment]
Autopan, vinylbreak, the filters... 99% of the time I use the mod delay, sometimes chorus and the shifters ... I was wondering would it be possible to manipulate the output compression outside the mfx, somewhere in the system  ? To have the option to compres the sound and use some nice stereo fx at the same time… a stereo reverb for the IFX would be a nice add for me…
